lib/services/swagger-api-service.js

```diff
@@ -153,12 +153,14 @@ function swaggerFactory(
             res.set('Content-Type', 'application/json');
             if (_.isArray(res.body)) {
                 render = Promise.map(res.body, function(element) {
+                    element.basepath = req.swagger.operation.api.basePath;
```

Comment:
Nevermind.  It looks like `element` has already been stripped of its instance methods, so this is fine as is.